docs/src/benchmark.md

```diff
@@ -0,0 +1,99 @@
+With a JSO-compliant solver, such as CaNNOLeS, we can run the solver on a set of problems, explore the results, and compare to other JSO-compliant solvers using specialized benchmark tools. 
+We are following here the tutorial in [SolverBenchmark.jl](https://juliasmoothoptimizers.github.io/SolverBenchmark.jl/v0.3/tutorial/) to run benchmarks on JSO-compliant solvers.
+``` @example ex1
+using NLSProblems, NLPModels
+```
+
+To test the implementation of CaNNOLeS, we use the package [NLSProblems.jl](https://github.com/JuliaSmoothOptimizers/NLSProblems.jl), which implements `NLSProblemsModel` an instance of `AbstractNLPModel`. 
+
+``` @example ex1
+using SolverBenchmark
+```
+
+Let us select equality-constrained problems from NLSProblems with a maximum of 10000 variables or constraints. After removing problems with fixed variables, examples with a constant objective, and infeasibility residuals, we are left with 82 problems.
+
+``` @example ex1
+problems = (NLSProblems.eval(problem)() for problem in filter(x -> x != :NLSProblems, names(NLSProblems)) )
+```
+
+We compare here CaNNOLeS with `trunk` (A. R. Conn, N. I. M. Gould, and Ph. L. Toint (2000). Trust-Region Methods, volume 1 of MPS/SIAM Series on Optimization.) implemented in [JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) on a subset of NLSProblems problems.
+``` @example ex1
+using CaNNOLeS, JSOSolvers
+```
+ To make stopping conditions comparable, we set `Ipopt`'s parameters `dual_inf_tol=Inf`, `constr_viol_tol=Inf` and `compl_inf_tol=Inf` to disable additional stopping conditions related to those tolerances, `acceptable_iter=0` to disable the search for an acceptable point.
```

Comment:
Right above it was `trunk`, now it's Ipopt and below is `tron` and `trunk`.

Reply:
Thanks! I fixed that one.